juju/juju
# [JUJU-1976] Upgrades Juju to latest description package.

With this change Juju is now using the latest juju/description package and because of this needs to be change to support the series removal from description.

juju/description also allows for model migrations between 2.8 and 2.9.

Fixes LP1926337

## Checklist

*If an item is not applicable, use `~strikethrough~`.*


## QA steps

1. Bootstrap a microk8s cluster with juju 2.8
2. Add a new model
3. Bootstrap this PR to the same microk8s cluster under a different name
4. Run a model migration from the first bootstrap controller model to the second controller.

Confirm there are no errors.

## Documentation changes

N/A

## Bug reference

https://bugs.launchpad.net/juju/+bug/1926337
